### ISSUE TITLE
[FEATURE] Bug Up block placement

### ISSUE THREAD

**Issue by superblaubeere27** [CCBlueX/LiquidBounce1.8-Issues#3875]
If you fall down next to a wall, BugUp will attempt to place a block, that safes you. Useful for e.g. CubeCraft

**Comment by Teals53** [comment]
well if you fall down next to a wall, scaffold or tower can safes you too just press your scaffold or tower bind. Also useful for everywhere xD

**Comment by CzechHek** [comment]
> well if you fall down next to a wall, scaffold or tower can safes you too just press your scaffold or tower bind. Also useful for everywhere xD

scaffold usually places blocks around you and doesn't save you

**Comment by Litely1222** [comment]
> > well if you fall down next to a wall, scaffold or tower can safes you too just press your scaffold or tower bind. Also useful for everywhere xD
> 
> scaffold usually places blocks around you and doesn't save you

**Comment by Teals53** [comment]
so if i can use this like a passive it can be useable so no longer i need to press bind keys for save my life it can be do this automatically and always stay

**Comment by Litely1222** [comment]
sometime it flag

**Comment by superblaubeere27** [comment]
> 
> 
> well if you fall down next to a wall, scaffold or tower can safes you too just press your scaffold or tower bind. Also useful for everywhere xD

Sometimes AutoBlock is disabled, sometimes you don't know that you will die if you don't activate scaffold, sometimes, you are too slow for it.

**Comment by Teals53** [comment]
> > well if you fall down next to a wall, scaffold or tower can safes you too just press your scaffold or tower bind. Also useful for everywhere xD
> 
> Sometimes AutoBlock is disabled, sometimes you don't know that you will die if you don't activate scaffold, sometimes, you are too slow for it.

yeah good idea also i am soo lazy for take it my mlg water back after fall xD

**Comment by MaxHD00** [comment]
And for sprint jumping, the fallprotect from the GommeScaffold script can help. It can save you from sprint jumping.
